src/main/java/frc/utils/battery/BatteryLimiter.java

```diff
@@ -20,43 +21,50 @@ class BatteryLimiter extends Command {
 
 	private final BooleanEntry lowBatteryEntry;
 	private final LinearFilter voltageFilter;
+	private double averageVoltage;
 
 	public BatteryLimiter() {
 		this.voltageFilter = LinearFilter.movingAverage(NUMBER_OF_SAMPLES_TAKEN_IN_AVERAGE);
+		this.averageVoltage = BatteryUtils.getCurrentVoltage();
 		this.lowBatteryEntry = NetworkTableInstance.getDefault().getBooleanTopic(LOW_BATTERY_TOPIC_NAME).getEntry(false);
 		lowBatteryEntry.set(false);
 
+		AlertManager.addAlert(
+			new PeriodicAlert(
+				Alert.AlertType.WARNING,
+				BatteryConstants.LOG_PATH + "LowVoltageAt",
+				() -> averageVoltage <= BatteryUtils.MIN_VOLTAGE
+			)
+		);
+
 		if (Robot.ROBOT_TYPE.isSimulation()) {
 			CMDHandler.runPythonClass(Path.of("BatteryMessage"), IPs.SIMULATION_IP);
 		}
 	}
 
+	private void startVoltageFilter() {
```

Comment:
```suggestion
	private void resetVoltageFilter() {
```

Reply:
Id say set and not reset because you are not erasing anything.

Reply:
reset is set again, but set is fine too